jcl/source/windows/JclDebug.pas

```diff
@@ -3498,6 +3545,73 @@ function TJclBinDebugScanner.SourceNameFromAddr(Addr: DWORD): string;
     Result := '';
 end;
 
+function TJclBinDebugScanner.VAFromUnitAndProcName( UnitName, ProcName : string ): DWORD;
+var
+  P          : Pointer;
+  VA         : Cardinal;
+  i,
+  Value,
+  FirstWord,
+  SecondWord : Integer;
+  S          : string;
+begin
+  Result := 0;
+  if ( UnitName = '' ) or ( ProcName = '' ) then
+    Exit;
+  ProcName := UnitName + '.' + ProcName;
+
+  if FCacheData then
+    begin
+    CacheProcNames;
+    for i := Low( FProcNames ) to High( FProcNames ) do
+      begin
+      if ( FProcNames[ I ].Text <> '' ) then
+        S := FProcNames[ I ].Text
```

Comment:
The `else` part has a `begin/end` block, so the `if` part must also have one and this line then takes a final semi colon